namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-pin-phone-monitor-prod/resources/iam.tf

```diff
@@ -0,0 +1,59 @@
+resource "aws_iam_user" "bt_upload_user" {
+  name = "hmpps-pin-phone-monitor-prod-bt-upload-user"
+  path = "/system/hmpps-pin-phone-monitor-prod-upload-users/"
+}
+
+resource "aws_iam_access_key" "bt_upload_user_key" {
+  user = aws_iam_user.bt_upload_user.name
+}
+
+data "aws_iam_policy_document" "bt_upload_policy" {
+  statement {
+    actions = ["s3:PutObject"]
+
+    resources = [
+      "${module.hmpps_pin_phone_monitor_document_s3_bucket.bucket_arn}/*.mp3",
+      "${module.hmpps_pin_phone_monitor_document_s3_bucket.bucket_arn}/*.flac",
+      "${module.hmpps_pin_phone_monitor_document_s3_bucket.bucket_arn}/*.wav",
+      "${module.hmpps_pin_phone_monitor_document_s3_bucket.bucket_arn}/*.xml",
```

Comment:
Maybe get rid of xml? We wouldn't be able to process any xml files so stop them sending entirely?